common/lib/capa/capa/templates/formulaequationinput.html

```diff
@@ -28,6 +29,6 @@
   <div class="script_placeholder" data-src="${previewer}"/>
 
   % if msg:
-      <span class="message">${msg|n}</span>
+      <span class="message">${HTML(msg)}</span>
```

Comment:
Nit: in general, it is advisable to name the variable something that makes it clear that it is HTML instead of plain text. But I'm OK with not touching the additional files.